docs/installation.md

```diff
@@ -78,7 +78,33 @@ If you want to help develop Lychee, install the development dependencies by remo
 
 #### Directory Permissions
 
-Directories within the `storage` and the `bootstrap/cache` directories must be writable by your web server or Lychee will not run. Additionally, `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
+Permissions of certain directories must be set correctly for Lychee to run.
+Note that there are up to three OS users which need to be taken into consideration:
+
+ - The user who is running the Web server daemon; called the **Web user** in the following.
+   Depending on your distribution and web server typically this user is called `apache`, `nginx`, `www` or `www-data`.
+ - (Optional) The user who is running the PHP FPM (or PHP CGI) pool; called the **PHP user** in the following.
+   Not every setup uses PHP FPM or PHP CGI.
+   For example, Apache executes the PHP interpreter as part of its own process if the extension `mod_php` is used.
+   In case PHP FPM is used, for most distributions PHP FPM or CGI ships with a sensible default configuration which ensures that the PHP user is the same as the Web user.
+   In both cases the PHP user is not of concern.
+   However, we are at least aware of one exception to this rule for Nginx on Fedora, see [FAQ](https://lycheeorg.github.io/docs/faq.html#i-know-my-file-permissions-for-storage-bootstrapcache-publicuploads-and-publicdist-are-correct-and-accessible-by-my-web-server-user-but-im-still-getting-a-php-error-when-writing-to-any-of-these-directories)
+ - The user which you use for shell logins and run scripts; called the **CLI user** in the following.
+   This user may of particular concern, if you are planning to upload photos via the web interface _and_ import photos via the shell scripts.
+
+The following permissions must be granted at least:
```

Comment:
I noticed this whole list is just PHP+Web, so it would save repeating it. I'm not sure how well it would fit with the "if you also..." part though, so feel free to ignore this.
```suggestion
At a minimum, the following permissions must be granted to the Web and PHP users:
```

Reply:
I am fine with putting "At a minimum" upfront. But I am not happy with adding "Web and PHP users", because this is the heading for the entire list and the list also contains statement about the CLI user (last entry).

Reply:
"At a minimum" was just there to make the end fit better, and yeah, I saw that after the fact. I feel it might be able to be trimmed but I'm not in the headspace for that right now and it's good enough:tm: